package.json

```diff
@@ -60,9 +60,18 @@
   "author": "Juan Cruz Viotti <juan@balena.io>",
   "license": "Apache-2.0",
   "engines": {
-    "node": ">=6.0"
+    "node": ">=8.0"
```

Comment:
Yes, good point, I'll create a separate commit. 👍  The Node version bump was because of oclif, by the way. There is another open PR updating the docs for other reasons, which I will update to make Node 8 a requirement: https://github.com/balena-io/balena-cli/pull/1175

Reply:
Now there's a separate commit: _"Bump min Node.js version to 8.0, ts-node to 8.1 and typescript to 3.4."_ I've taken the opportunity of the major change type on a separate commit to bump `ts-node` and `typescript` as well. The bump to `ts-node` required (or highly benefited from) a refactoring of the `typings` folder for use with the tsconfig `typeRoots` compiler option. It allows faster ts-node start-up time because it allows ts-node to skip the tsconfig `include` instruction, and allows loading typing definitions on demand rather than all at once before the application starts. With ts-node v8.1.0, running `./bin/balena-dev version` takes ~8 seconds on my laptop with the old typings configuration, and just under 3 seconds with the new typings configuration. (The compiled version takes about 1 second.)